src/ClassApplication/Services/DBService.cs

```diff
@@ -0,0 +1,112 @@
+using ACMTTU.NoteSharing.Shared.SDK.Services;
+using Microsoft.Azure.Cosmos;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using ACMTTU.NoteSharing.Shared.SDK.Controllers;
+using ACMTTU.NoteSharing.Platform.NotesApplication.Services;
+using NotesApplication;
```

Comment:
lol i know i said it should just be a copy pasta, but remember this is for the class application XD might need some slight changes...